src/test.js

```diff
@@ -272,12 +272,9 @@ Test.prototype = {
 				runtime: now() - this.started
 			};
 
-		if ( !result ) {
-			source = sourceFromStacktrace();
-
-			if ( source ) {
-				details.source = source;
-			}
+		source = sourceFromStacktrace();
```

Comment:
You can pass a parameter to `sourceFromStacktrace` to set the exact line you need to cap end send. The default (no parameter) works fine when the result is falsy, but it's not the same when it pass.

The example below - of the tests running with this current changes - seems we need to set the parameter to start with that second line (` at file:///Users/leobalter/dev/qunit/test/logs.js:120`).

Also, it's probably good to break this information to give just a single line location.

```
"    at temp//@1_Users@1_leobalter@1_dev@1_qunit@1_dist@1_qunit.js:9
     at file:///Users/leobalter/dev/qunit/test/logs.js:120
     at temp//@1_Users@1_leobalter@1_dev@1_qunit@1_dist@1_qunit.js:9
     at temp//@1_Users@1_leobalter@1_dev@1_qunit@1_dist@1_qunit.js:9
     at process (temp//@1_Users@1_leobalter@1_dev@1_qunit@1_dist@1_qunit.js:9)
     at begin (temp//@1_Users@1_leobalter@1_dev@1_qunit@1_dist@1_qunit.js:9)
     at temp//@1_Users@1_leobalter@1_dev@1_qunit@1_dist@1_qunit.js:9"
```